.github/workflows/playwright.yml

```diff
@@ -0,0 +1,28 @@
+name: Playwright
+on:
+    push  
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+    - name: Setup yarn
+      run: npm install -g yarn
+    - name: Install dependencies
+      run: yarn install --frozen-lokfile
```

Comment:
Invalid option.

**suggestion:** You can refer to an existing GitHub Action to be able to install project dependencies.

see: https://github.com/ovh/manager/blob/21eac231d2517524ba91a3dc88b5c5e163156aa0/.github/workflows/linter.yaml#L13-L28

Reply:
Refactored to use suggested existing syntax.